Jenkinsfile_CNP

```diff
@@ -167,29 +169,30 @@ withPipeline(type, product, component) {
 
   onPR {
     env.ENVIRONMENT = "preview"
-    env.CASE_API_URL = "http://privatelaw-hearings-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.CASE_API_URL = "https://fis-hmc-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.CASE_API_TEST_URL = "https://fis-hmc-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
```

Comment:
env.CASE_API_TEST_URL. need. to add onMaster ,demo,...

Reply:
done